### PR TITLE
Components

### PR DESCRIPTION
**Pull request by MAleemH**
I have done the following things:

- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches the given design.
- Just the UI of the calculator is developed.
- Functional components are being used.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <p className="keys">2</p>
+    <p className="keys">3</p>
+    <p className="keys right-col">+</p>
+    <p className="bottom-zero">0</p>
+    <p className="keys">.</p>
+    <p className="keys right-col">=</p>
+  </div>
+);
+
+export default Calculator;
```
Comment (julie-ify) [inline src/components/calculator.js]: - [ ] The Calculator should have at least one child component. Kindly implement that.

  Perhaps you can make these buttons into one reusable component and import it here.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @MAleemH,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] Kindly delete the following files.

  owpq
  oas

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (MAleemH) [reply src/components/calculator.js]: Hello @julie-ify, I think I'm not supposed to delete reportwebvitals.js and setuptests.js files. As I delete them my program doesn't run and even this is not mention in requirements.

**Review by TadesseDev — APPROVED** [review]
### Hi @MAleemH 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**